redbot/cogs/mod/names.py

```diff
@@ -69,6 +69,96 @@ class ModInfo(MixinMeta):
             else:
                 await ctx.send(_("Done."))
 
+    def handle_custom(self, user):
+        a = [c for c in user.activities if c.type == discord.ActivityType.custom]
+        if not a:
+            return None, discord.ActivityType.custom
+        a = a[0]
+        c_status = None
+        if not a.name and not a.emoji:
+            return None, discord.ActivityType.custom
+        elif a.name and a.emoji:
+            c_status = _("Custom: {emoji} {name}").format(emoji=a.emoji, name=a.name)
+        elif a.emoji:
+            c_status = _("Custom: {emoji}").format(emoji=a.emoji)
+        elif a.name:
+            c_status = _("Custom: {name}").format(name=a.name)
+        return c_status, discord.ActivityType.custom
+
+    def handle_playing(self, user):
+        p_acts = [c for c in user.activities if c.type == discord.ActivityType.playing]
+        p_act = p_acts[0] if p_acts else None
+        act = (
+            _("Playing: {name}").format(name=p_act.name if p_act and p_act.name else None)
+            if p_act and hasattr(p_act, "name")
+            else p_act.name
+            if p_act and p_act.name
+            else None
+        )
```

Comment:
This seems overcomplicated - activities always have a `name` attribute. Unless there's more to it than I see, you should be able to just simplify it to:
```suggestion
        if not p_acts:
            return
        p_act = p_acts[0]
        act = _("Playing: {name}").format(name=p_act.name)
```